src/main/java/nl/workingtalent/bookrental/controller/UserController.java

```diff
@@ -114,8 +116,13 @@ public String userLogin(@RequestBody LoginDto loginDto) {
 		String token = RandomStringUtils.random(150, true, true);
 		foundUser.setToken(token);
 		userRepo.save(foundUser);
-					
-		return token;
+		Map<String, String> map = new HashMap<String, String>();	
```

Comment:
Misschien wat comments toevoegen over wat de verschillende returns zijn. Voor zover ik het nu begrijp probeer je een Dictionary te returnen die zowel de inlogtoken en of het een admin is ja/nee bevat. Maar in dat geval is een boolean misschien logischer, en het is redelijk verwarrend waarom je map.put("isAdmin", ""); doet.

Reply:
Goeie, 
dit moest bij een eerdere oplossing omdat er toen geen String en Boolean gecombineerd konden worden.
Ik zal er comments bij zetten, maar de boolean op de to-do lijst zetten voor een volgende commit, omdat dan de front-end ook aangepast moet worden en dan kunnen jullie later pas verder